docs/DeveloperGuide.md

```diff
@@ -137,26 +138,30 @@ How the parsing works:
 ### Model component
 **API** : [`Model.java`](https://github.com/AY2122S1-CS2103T-W08-3/tp/tree/master/src/main/java/seedu/address/model/Model.java)
 
-<img src="images/ModelClassDiagram.png" width="450" />
+<img src="images/ModelClassDiagram.png" width="550" />
 
+Class Diagram of the `Person`'s entity:
 
-The `Model` component,
+<img src="images/PersonClassDiagram.png" />
 
-* ModelManager class stores these four components.
-  * Address book data i.e., all `Person` objects (which are contained in a `UniquePersonList` object).
-  * Task book data i.e., all `Tasks` objects (which are contained in a `TaskList` object).
-  * Order book data i.e., all `Orders` objects (which are contained in a `OrderList` object).
-  * `UserPref` object that represents the user’s preferences. This is exposed to the outside as a `ReadOnlyUserPref` objects.
+Class Diagram of the `Task`'s entity:
 
-* stores the currently 'selected' `Person` objects (e.g., results of a search query) as a separate _filtered_ list which is exposed to outsiders as an unmodifiable `ObservableList<Person>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
-* does not depend on any of the other three components (as the `Model` represents data entities of the domain, they should make sense on their own without depending on other components)
+<img src="images/TaskClassDiagram.png" />
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `SalesNote`, which `Person` references. This allows `SalesNote` to only require one `Tag` object per unique tag, instead of each `Person` needing their own `Tag` objects.<br>
+Class Diagram of the `Order`'s entity:
 
-<img src="images/BetterModelClassDiagram.png" width="450" />
+<img src="images/OrderClassDiagram.png" />
 
-</div>
+The `Model` component,
+
+* stores these three types of data in SalesNote
+  * Address book data i.e., all `Person` objects (which are contained in a `UniquePersonList` object).
+  * Task book data i.e., all `Tasks` objects (which are contained in a `UniqueTaskList` object).
```

Comment:
Using `Task` and `Order` in this section instead of `Tasks` and `Orders` might be more consistent